### DISCUSSION TITLE
Create an Application

### DISCUSSION THREAD

**Discussion by selarem** [idaholab/moose#23656] [Q&A Getting Started] [answered]
Hello,
I am creating a new app. In the Makefile I have put yes for all modules. It compiles fine.
However, when I tried to run the crack2d_vol_dev.i of the  phase_field_fracture, I got:
section '[Modules/PhaseField]' does not have an associated "Action".
 Common causes:
- you misspelled the Action/section name
- the app you are running does not support this Action/syntax

section '[Modules/PhaseField/Nonconserved]' does not have an associated "Action".
 Common causes:
- you misspelled the Action/section name
- the app you are running does not support this Action/syntax

Normally, with yes for all modules, shouldn't i be able to run all the tests ?

thanks in advance for any help.

**Comment by GiudGiud** [comment, marked as answer]
Hello

This should have worked
Could you please paste your Makefile here?
Do you have `PHASE_FIELD                 := yes`
And did you recompile after?

Guillaume

**Comment by selarem** [comment]
Everything is yes.
But It des not work.

**Reply by GiudGiud** [reply]
when it compiles do you see the phase field module build?
If you run <my_exec-opt>  --dump do you see the phase field syntax?

This should work, unless you are using an old executable from before you set it to yes

**Reply by selarem** [reply]
Yes I see PhaseFieldFracture syntax.

**Reply by GiudGiud** [reply]
I just installed a new app from scratch and got it to run that input. 

Please try installing from scratch again (at least git clean -xfd in your app and the moose folder)
Take care, git clean removes all NON-COMMITTED work

**Reply by selarem** [reply]
It works after reinstall of moose. It should not be like this.
thanks

**Reply by GiudGiud** [reply]
If you can make the issue reproducible we can attempt to fix it. 
i could not get the same problem
